solc/jsonCompiler.cpp

```diff
@@ -58,23 +58,35 @@ ReadFile::Callback wrapReadCallback(CStyleReadFileCallback _readCallback = nullp
 	{
 		readCallback = [=](string const& _path)
 		{
+			ReadFile::Result result;
 			char* contents_c = nullptr;
 			char* error_c = nullptr;
-			_readCallback(_path.c_str(), &contents_c, &error_c);
-			ReadFile::Result result;
-			result.success = true;
+			try
```

Comment:
Not sure it is possible to have an exception here given `_readCallback` is a C function?